msgraph/generated/admin/admin_request_builder.py

```diff
@@ -32,7 +31,7 @@ def __init__(self,request_adapter: RequestAdapter, path_parameters: Union[str, D
         """
         super().__init__(request_adapter, "{+baseurl}/admin{?%24expand,%24select}", path_parameters)
     
-    async def get(self,request_configuration: Optional[RequestConfiguration] = None) -> Optional[Admin]:
+    async def get(self,request_configuration: Optional[AdminRequestBuilderGetRequestConfiguration] = None) -> Optional[Admin]:
```

Comment:
This looks like it will resolve the changes we had breaking in 1.3.0

Reply:
it would, but it'd also break the people who migrated to the new RequestConfiguration

Reply:
The syntax we shared would have worked for both V1.2.0 and V 1.3.0 - so it might not break as this would be similar to 1.2.0